behavioral-questions.md

```diff
@@ -4,16 +4,28 @@ Answer the following questions. Use the Nugget + STAR method for each answer.
 
 1. Why did you choose a career in software development?
 
-- Nugget:
-- Situation:
-- Task:
-- Action:
-- Result:
+- Nugget: Ive dabbled a bit in technological concepts like networking and pc building. It wasnt until i got out of the military that I noticed how powerful having knowledge in technical skills like networking can be in the real world. Then with some research I stumbled across the vettec that offered two programs I was intersted in my area. both of which offered learning software development. Some more research insued and I grew very interested in pursuing a career in software development.
+
+- Situation: I grasped some concepts in networking pretty well but when it came to learning any programming language, I tried out things like udemy, youtube, and other online tools to no avail.
+
+- Task: I needed a more structured learning environment that I would be more dynamic and would also hold me accountable to keep me from straying off my learning. Its very easy to let trivial things in life to get in the way of your learning when there is nothing holding you accountable.
```

Comment:
If you were responding to this question in an interview, be careful about exposing possible weaknesses when not prompted.  Perhaps there is a way to rephrase some of this that focuses on the positives of you and software development.